alchemist/alchemist-cognitive-agents/src/main/kotlin/it/unibo/alchemist/model/cognitiveagents/characteristics/individual/Age.kt

```diff
@@ -12,10 +8,23 @@ enum class Age : IndividualCharacteristic {
 
     companion object {
 
+        private const val CHILD_THRESHOLD = 18
+        private const val ADULT_THRESHOLD = 60
+        private val CHILD_KEYWORDS = setOf("child", "CHILD")
```

Comment:
Use `equalsIgnoreCase` rather than a set. It's much better, coincise, and supports cases such as "Child".

Reply:
good to know